brianlong/www.validators.app
# 182557882_upgrade_to_bootstrap_5

#### What's this PR do?
-upgrade to bootstrap 5

#### How should this be manually tested?
- To test locally: remove cached assets first: `rm -r node_modules public/packs public/assets`
- Make sure you use correct version of node (if you're using nvm to manage node versions then you can type in `nvm use` and it should pick up correct version automatically). Then run `yarn install`
- Click through the app on desktop and mobile and look for any errors, 500, not working js etc.

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/182557882)
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/n/projects/2177859/stories/183010241)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
